tools/packaging/packaging.psm1

```diff
@@ -940,15 +940,15 @@ function New-MacOsDistributionPackage
     try
     {
         # productbuild is an xcode command line tool, and those tools are installed when you install brew
-        Start-NativeExecution -sb {productbuild --distribution $distributionXmlPath --resources $resourcesDir $newPackagePath}
+        Start-NativeExecution -sb {productbuild --distribution $distributionXmlPath --resources $resourcesDir $newPackagePath} > $null
```

Comment:
There is a parameter that will suppress output unless there is an error.

Reply:
Fixed. Used `-VerboseOutputOnError`